odoo/addons/base/models/ir_actions_report.py

```diff
@@ -287,14 +287,20 @@ def _build_wkhtmltopdf_args(
             elif paperformat_id.header_spacing:
                 command_args.extend(['--header-spacing', str(paperformat_id.header_spacing)])
 
-            command_args.extend(['--margin-left', str(paperformat_id.margin_left)])
+            if specific_paperformat_args and specific_paperformat_args.get('data-report-margin-left'):
+                command_args.extend(['--margin-left', str(specific_paperformat_args['data-report-margin-left'])])
+            else:
+                command_args.extend(['--margin-left', str(paperformat_id.margin_left)])
```

Comment:
the new options are being added as the existing ones are, but since this code is so similar maybe we could do:

```suggestion
            if specific_paperformat_args is None:
                specific_paperformat_args = {}
            for param in ('header-spacing', 'margin-top', 'margin-bottom', 'margin-left', 'margin-bottom'):
                value = specific_paperformat_args.get('data-report-' + param) or paperformat_id[param.replace('-', '_')]
                command_args.extend(['--' + param, str(value)])
```
and remove the corresponding 20 lines of code (also if `if ...: specific_paperformat_args = {}` was put at the top of the method, we could also remove 2 `specific_paperformat_args and`)

it's not necessary to do and the current version is ok and follows the existing style, but the current code is rather mouthy for how repetitive it is